packages/core-bridge/src/lib.rs

```diff
@@ -202,6 +205,7 @@ where
     Ok(())
 }
 
+/// Call [callback] with an UnexpectedError created from [err]
```

Comment:
`[foo]` is for doc links, args can't be linked to, so these should just be surrounded with `